src/components/Form/ServerPaginatedDDList.js

```diff
@@ -120,7 +129,7 @@ const ServerPaginatedDDList = (props) => {
 
     useEffect(() => {
         /* searh query changed -> reset page no. to 1 */
-        pageNoRef.current = 1;
+        pageNoRef.current = getPageNoRef();
```

Comment:
A better name for this would be `getDefaultPageNo` as we are not returning any ref but a static value